src/main/java/com/example/bookstore/controller/users/CartsController.java

```diff
@@ -55,4 +56,32 @@ public String showCart(@RequestParam(value = "userId", required = false) Integer
 		return "users/cart";
 	}
 
+	@PostMapping("/cart/update")
```

Comment:
```suggestion
	@PostMapping("/carts/{id}")
```
mà update thì chị thấy phairt PATCH chứ nhỉ